sdk/keyvault/azure-keyvault-administration/azure/keyvault/administration/_settings_client.py

```diff
@@ -35,7 +35,7 @@ def get_setting(self, name: str, **kwargs) -> KeyVaultSetting:
 
         :returns: The account setting, as a :class:`~azure.keyvault.administration.KeyVaultSetting`.
         :rtype: ~azure.keyvault.administration.KeyVaultSetting
-        :raises: :class:`~azure.core.exceptions.HttpResponseError`
+        :raises: azure.core.exceptions.HttpResponseError
```

Comment:
Does this render correctly? IIRC if the colon goes after the type then the exception type is clickable, like

```suggestion
        :raises azure.core.exceptions.HttpResponseError:
```

Reply:
You're right about that -- I can update our `:raises:` fields to do this. Do you know if there's a similar method for methods that have multiple exceptions that can be raised?

Reply:
You can separate them with `or`:

`:raises ~azure.core.exceptions.HttpResponseError or ~azure.core.exceptions.ResourceNotFoundError:`